x-pack/legacy/plugins/apm/server/lib/transactions/distribution/get_buckets/fetcher.ts

```diff
@@ -54,7 +54,7 @@ export function bucketFetcher(
         distribution: {
           histogram: {
             field: TRANSACTION_DURATION,
-            fixed_interval: bucketSize,
+            interval: bucketSize,
```

Comment:
If we had better typescript support, this could probably have been caught. Eg. `histogram` doesn't have a `fixed_interval` prop.

Reply:
Yeah, that's my thinking as well. I think we can add them w/ the new aggregation types.